### PR TITLE
Improvements to VTOL and FlightCore guide 

### PR DESCRIPTION
**Pull request by itscynxx**
Changes the quotes in the VTOL guide to be asterisks, as well as make the image for right clicking a mod display properly on gitbook.

Fixes grammar in the FlightCore guide's Settings section

### CODE REVIEW

**Review by GeckoEidechse — CHANGES_REQUESTED** [review]
Two things
1. Markdown allows both underline (`_`) and single asterisk (`*`) for italics but GitBook prefers underscore. It still accepts asterisk just fine but when editing something on GitBook it will update all those lines to use underscore so might as well use asterisk from the beginning
2. I'm fine with letting it slide this time but generally make sure to keep separate changes in separate PRs cause it makes it easier to revert them individually if need be and allows for merging parts that are fine early.
  For example the changes to the FlightCore guide are all fine and I could've already merged it if it weren't for stuff in the (unrelated) VTOL guide holding it back :P

**Comment by GeckoEidechse** [comment]
Also forgot to mention but I'd use both quotation marks and italics when talking about button names etc, i.e.

> `_"Example button name"_`

which gives

> _"Example button name"_

This is because italics on its own can be hard to detect for people with poor eye sight.

**Comment by itscynxx** [comment]
I'll try to split them from now on, just didn't want to spam pr's 

Will add the underscores now

**Comment by GeckoEidechse** [comment]
> I'll try to split them from now on, just didn't want to spam pr's

haha, yeah I get the concern and yet as counter-intuitive as it might seems it's actually the preferred approach as smaller individual changes are a lot easier (and therefore faster) to review. So spam away as much as you want <3

File: docs/installing-northstar/northstar-installers/vtol-guide.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ VTOL is a mod manager for installing and managing mods for the Northstar client
 
 ![VTOL Main Window](../../images/vtol-main-window.png)
 
-**The *Main Menu* of VTOL is where you'll probably spend most of your time.**
+__The _Main Menu_ of VTOL is where you'll probably spend most of your time.__
```
Comment (GeckoEidechse) [inline docs/installing-northstar/northstar-installers/vtol-guide.md]: Pretty sure you intended that line to still stay bold? I.e. it should be

```suggestion
**The _Main Menu_ of VTOL is where you'll probably spend most of your time.**
```

?

File: docs/installing-northstar/northstar-installers/vtol-guide.md [hunk 2]
```diff
@@ -37,13 +37,13 @@ VTOL handles most things automatically, but it may still require minimal setup f
...
 
 ## Launching Northstar via VTOL
 
 ![Launching Northstar](../../images/vtol-launching-northstar.png)
 
-Assuming you've followed everything up to this point, you should be set to hit "Launch Titanfall 2 Northstar" and play on Northstar's servers. If you encounter an error like a crash, Northstar will create a log file in the "titanfall2/r2northstar/logs" directory. You can look at this log yourself, or send it on the [Northstar Discord server](https://discord.com/invite/northstar) and someone can try to look at it and help you.
+Assuming you've followed everything up to this point, you should be set to hit _"Launch Titanfall 2 Northstar"_ and play on Northstar's servers. If you encounter an error like a crash, Northstar will create a log file in the `titanfall2/r2northstar/logs` directory. You can look at this log yourself, or send it on the [Northstar Discord server](https://discord.com/invite/northstar) and someone can try to look at it and help you.
```
Comment (GeckoEidechse) [inline docs/installing-northstar/northstar-installers/vtol-guide.md]:
```suggestion
Assuming you've followed everything up to this point, you should be set to hit _"Launch Titanfall 2 Northstar"_ and play on Northstar's servers. If you encounter an error like a crash, Northstar will create a log file in the `Titanfall2/R2Northstar/logs` directory. You can look at this log yourself, or send it on the [Northstar Discord server](https://discord.com/invite/northstar) and someone can try to look at it and help you.
```

might as well also fix capitalisation while at it :P

File: docs/installing-northstar/northstar-installers/vtol-guide.md [hunk 3]
```diff
@@ -56,56 +56,57 @@ To expand the smaller version of the tab list, press the button with 3 bars as s
 
 ![3 Bars](../../images/vtol-settings-expand.png)
 
-**NOTE: This guide will not be including the "Home" button as a tab when describing this list. When it says the first tab, it means the first tab underneath the home button**
+__NOTE: This guide will not be including the _"Home"_ button as a tab when describing this list. When it says the first tab, it means the first tab underneath the home button__
```
Comment (GeckoEidechse) [inline docs/installing-northstar/northstar-installers/vtol-guide.md]:
```suggestion
**NOTE: This guide will not be including the _"Home"_ button as a tab when describing this list. When it says the first tab, it means the first tab underneath the home button**
```

Again I assume you wanted that line to still stay bold? Double underscore is not properly defined in vanilla markdown IIRC. Discord uses it for underlining a sentence but not sure if that also applies to GitBook...

**Review by GeckoEidechse — APPROVED** [review]
Requested changes have been addressed. Looked over the full diff again and looks good to me now.

**Pull request merged**